onset_maproom/layout.py

```diff
@@ -183,6 +183,31 @@ def controls_layout():
                 agricultural campaign associated with it.
                 """
             ),
+            Block("Latitude / longitude inputs",
+                dbc.Row(
+                    [
+                        dbc.Col(
+                            dbc.Input(
+                                id = "latInput", 
+                                placeholder="Type latitude...", 
```

Comment:
what about putting the allowed range here (3-15)?
And what about forcing out of range entries back in the range? ie if someone enters 2, we se to 3?
These are open questions and I think should go to another PR if we want to address them as there's already a lot in there.

Reply:
This made me realize that there are no lat/lon ticking on our map. I wonder how much it matters now that we have the ability to both click and type-in...

Reply:
As in, put the allowed range as a text within the input box instead of the 'type latitude...' ?

I can also look into seeing if we can auto-update the box as you say if you think that is better then just getting a red box if it is out of range. I suppose having the lat/lng tix boxes would be very nice in terms is visually choosing coordinates without having to click on the map and wait for the page to reload though.

Reply:
yes, when nothing is entered, show the acceptable range... That seems like a nice place to do it, and that would leave room for other info in the label below the box.
I didn't realize the red boxing when out of range. I remember seeing the red but having the impression it was always red... oh well... UX...

I was just thinking of having ticks which sounds appropriate for any figure in general to have ticks on the axes. Some sort of pop-up showing coordinates where your mouse is may be a useful feature too...